### PR TITLE
mysql-cluster minor fixes

### PR DESCRIPTION
**Pull request by bazmurphy**
here are some minor fixes for the `mysql-cluster` project that I noticed when going through it today

also because it popped up when i was trying to find the `Slave_IO_Running` and `Slave_SQL_Running` key:value pairs and couldn't see them, that i found out about:

> Starting from MySQL 8.0.22, the `MASTER` and `SLAVE` terminology in replication has been replaced with `SOURCE` and `REPLICA`. The old terms are still supported for backward compatibility, but the new terms are preferred.

So there are two changes to those key/values

But I also then adjusted some of the text to make it clearer that MASTER/SOURCE are the same and SLAVE/REPLICA are the same.

However this can read a bit weird, for example reading `REPLICATION REPLICA` in one of the queries is a bit unclear

@maksimov

### CODE REVIEW

File: projects/mysql-cluster/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Timebox: 2 days
...
 
-In this type of replication, the primary server (or the master) takes all the writes and they are automatically replicated onto the replica server (or the slave). This technique is widely used to increase the scalability of the database for read-intensive operations (which is extremely common for the web). In the primary-replica setup, the primary would normally be used for writes and replica (or replicas) for reads only. Even though it's technically possible to use the primary for the reads and the writes, it is impossible to write directly to the replica.
+In this type of replication, the primary server (or the source/master) takes all the writes and they are automatically replicated onto the replica server (or the replica/slave). This technique is widely used to increase the scalability of the database for read-intensive operations (which is extremely common for the web). In the primary-replica setup, the primary would normally be used for writes and replica (or replicas) for reads only. Even though it's technically possible to use the primary for the reads and the writes, it is impossible to write directly to the replica.
```
Comment (maksimov) [inline projects/mysql-cluster/README.md:24]: If `master` and `slave` are no longer in use in the queries, let's get rid of the terms completely

**Comment by bazmurphy** [comment]
@maksimov done
one small question what you want to have on line 22 
in the original brackets was (master-slave) 
could that be (source-replica) 
or as I have done at the moment to remove it

**Comment by maksimov** [comment]
@bazmurphy sorry for the delay. We internally call it "primary" (because of how Vitess calls it), but `SOURCE` is probably the best choice.

**Review by SallyMcGrath — APPROVED** [review]
ta!

**Pull request merged**